src/main/kotlin/day11/RuntimeError.kt

```
@@ -0,0 +1,10 @@
+package org.hyunjung.day11
+
+fun main() {
+    val numString = "10.5"
+    try {
+        println(numString.toInt())
+    } catch (e: RuntimeException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)